### ISSUE TITLE
Group versions of tasks

### ISSUE THREAD

**Issue by rwblair** [expfactory/expfactory-deploy#3]
- There are different but equally valid instantiations of an experiment (e.g. using a stop signal with a tone instead of a star) across groups as well as within groups. These should be clumped under standardized names (e.g. “stop signal”), and you should be able to choose the instantiation of the task by group/tag (e.g. poldrack__fmri, poldrack__network, ABCD [although obviously not the ABCD group’s stop signal haha], etc.)
- If this happens, it would be cool to link the high level task (e.g. stop signal) with the cognitive ontology for a description of the task and the constructs it’s associated with

**Comment by vsoch** [comment]
yes! We made a small effort to link to the cognitive atlas with `cognitive_atlas_task_id` https://github.com/expfactory/expfactory-experiments/blob/master/angling_risk_task/config.json#L22 and then could generate this "concept view" https://expfactory.github.io/v1/experiments.html and I agree it would be super cool to expand upon this.
